Error is thrown for make build-docker

<!--
Please fill in as much of the template below as you can.

Be ready for followup questions, and please respond in a timely
manner. We might ask you to provide additional logs and data (tendermint & app).
-->

**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):
unreleased-master-e4d6f6df096fe565b6eb70ac44c4f3873882ab64

**ABCI app** (name for built-in, URL for self-written if it's publicly available):

**Environment**:
- **OS** (e.g. from /etc/os-release): Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-88-generic x86_64)
- **Install tools**: Makefile
- **Others**:


**What happened**:
error is thrown after executing `make build-docker`

**What you expected to happen**:
successful build of docker image

**Have you tried the latest version**: yes

**How to reproduce it** (as minimally and precisely as possible):

1. Clone/pull latest master of tendermint
2. Execute `make build-docker`

**Logs (paste a small part showing an error (< 10 lines) or link a pastebin, gist, etc. containing more of the log file)**:
`make[1]: Entering directory '/home/nnv/go/src/github.com/tendermint/tendermint'
CGO_ENABLED=0 go build -mod=readonly -ldflags "-X github.com/tendermint/tendermint/version.TMVersion=unreleased-master-e4d6f6df096fe565b6eb70ac44c4f3873882ab64 -s -w " -trimpath -tags 'tendermint' -o /home/nnv/go/src/github.com/tendermint/tendermint/build/ ./cmd/tendermint/
/bin/sh: 1: go: not found
make[1]: *** [Makefile:68: build] Error 127
make[1]: Leaving directory '/home/nnv/go/src/github.com/tendermint/tendermint'
make: *** [Makefile:249: build-linux] Error 2
`

**Config (you can paste only the changes you've made)**: none

**node command runtime flags**:

**Please provide the output from the `http://<ip>:<port>/dump_consensus_state` RPC endpoint for consensus bugs**

**Anything else we need to know**:
on OSX, error of no docker-entrypoint.sh is thrown during execution of the same command

Thanks so much for the hint @creachadair 🙏 . Indeed my paths were gone somehow after the update for ubuntu 😓 
Nevertheless, here is the second error after fixing path
`Step 5/20 : RUN make build-linux
 ---> Running in 8deb589580c7
make: *** No rule to make target 'build-linux'.  Stop.
The command '/bin/sh -c make build-linux' returned a non-zero code: 2
make: *** [Makefile:230: build-docker] Error 2
`

I'm wondering to which of the Makefiles(inside docker dir or root of the repo) does the Dockerfile point to? 🧐

> Nevertheless, here is the second error after fixing path `Step 5/20 : RUN make build-linux ---> Running in 8deb589580c7 make: *** No rule to make target 'build-linux'. Stop. The command '/bin/sh -c make build-linux' returned a non-zero code: 2 make: *** [Makefile:230: build-docker] Error 2 `

I'm not sure where that error is coming from. However, before I dig too far into that—is your goal to build a Docker container explicitly, or are you just trying to get the `tendermint` binary? If you only need the latter, you can more simply run:

```
make build
```

--- a/Makefile
+++ b/Makefile
@@ -227,13 +227,13 @@ build-docs:
 
 build-docker: build-linux
 	cp $(BUILDDIR)/tendermint DOCKER/tendermint
-	docker build --label=tendermint --tag="tendermint/tendermint" DOCKER
+	docker build --label=tendermint --tag="tendermint/tendermint" -f DOCKER/Dockerfile .
 	rm -rf DOCKER/tendermint
 .PHONY: build-docker
 
 
 ###############################################################################
-###                       Mocks 											###
+###                                Mocks                                    ###
 ###############################################################################
 
 mockery:

